Makefile

```diff
@@ -45,7 +45,11 @@ install: gopath
 	install -D -m 00644 yum.conf.in $(DESTDIR)/usr/share/defaults/mixer/yum.conf.in
 
 check: gopath
-	go test ${GO_PACKAGE_PREFIX}/...
+	go test -cover ${GO_PACKAGE_PREFIX}/...
+
+checkcoverage: gopath
+	test ${PKG}
```

Comment:
You might like to know that in Go 1.10 we'll have support for passing multiple packages with a coverprofile, so no need to do pkg. :-)

https://github.com/golang/go/commit/283558e42b88a6afa39da6ad4ae87558dc053776

Optional: add a TODO so we eventually use that capability.